### ISSUE TITLE
[develop] Clicking on ZKN links does not perform global search anymore

### ISSUE THREAD

**Issue by kyaso** [Zettlr/Zettlr#1989]
## Description

I noticed in the current `develop` branch, that when clicking on ZKN links, no global search is performed. For tags it still works.

## Reproducing
Ctrl+click on any [[ZKN link]] --> target file should be opened, but no global search is started

## Expected behaviour
When clicking on a ZKN link, the target file should be opened and a global search for the ID (or whatever is between the [[ ]]) should be started (if enabled in options).

## Platform
 - OS and version: Manjaro Linux
 - Zettlr Version: `develop`

## Additional information
I looked through the code, and found this section in `Editor.vue`:
```vue
this.editor.on('zettelkasten-link', (linkContents) => {
      ipcRenderer.invoke('application', {
        command: 'force-open',
        payload: linkContents
      })
        .catch(err => console.error(err))
})
this.editor.on('zettelkasten-tag', (tag) => {
  this.$root.$emit('start-global-search', tag)
})
```

**Comment by nathanlesage** [comment]
I didn't (yet) re-implement that one, so keeping it open! Feel free to add that `start-global-search` thingy in a PR making sure it respects the config setting which you mentioned (and which I already have forgotten, shame on me!)

**Comment by nathanlesage** [comment]
Already fixed by #2003
